butlarr/services/sonarr.py

```diff
@@ -379,6 +405,10 @@ async def clbk_update(self, update, context, args, state):
                     menu=None,
                 )
                 full_redraw = True
+            elif len(args) > 1 and not is_int(args[1]):
+                state = replace(state, menu=args[1])
+                if args[1] in ("seasons", "episodes", "episode"):
+                    allow_edit = False
```

Comment:
What are you doing here? I dont' really get it tbh.
Why are you overloading goto like that?
Why not just a new command like goto_seasons or goto_episodes?

Reply:
I just needed allow edit to be false when changing the keyboard layout

> Why not just a new command like goto_seasons or goto_episodes?
That would probably work to, i'll rewrite that later

Reply:
Created new "goto_menu" command, and reverted the "goto" command to how it was :D

commit: https://github.com/TrimVis/butlarr/pull/15/commits/9fab45a40a51f98ea4205d398608197833d2dbe2